app/policies/additional_expense_policy.rb

```diff
@@ -0,0 +1,27 @@
+class AdditionalExpensePolicy < ApplicationPolicy
+  class Scope < ApplicationPolicy::Scope
+    def resolve
+      case user
+      when CasaAdmin, Supervisor
+        scope.joins([:case_contact, :casa_case]).where(casa_case: {casa_org_id: user.casa_org.id})
```

Comment:
Honestly am always confused about the array syntax here. Forget every time.  But I had to change this, so hold that thought... the relation doesn't work through CasaCases because CaseContacts don't have a casa_case_id until they are active.

Should have updates tomorrow.